### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by mhuggins** [romeovs/react-a11y#5]
Running `npm install romeovs/react-a11y --save` failed on my project.  This is what the output looks like:

> > react-a11y@1.0.0 postinstall /Users/mhuggins/Development/kapost/pepperweed/node_modules/react-a11y
> > npm run build
> 
> > react-a11y@1.0.0 build /Users/mhuggins/Development/kapost/pepperweed/node_modules/react-a11y
> > babel src --out-dir lib
> 
> sh: babel: command not found
> 
> npm ERR! Darwin 15.4.0
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
> npm ERR! node v5.11.0
> npm ERR! npm  v3.8.6
> npm ERR! file sh
> npm ERR! code ELIFECYCLE
> npm ERR! errno ENOENT
> npm ERR! syscall spawn
> npm ERR! react-a11y@1.0.0 build: `babel src --out-dir lib`
> npm ERR! spawn ENOENT
> npm ERR! 
> npm ERR! Failed at the react-a11y@1.0.0 build script 'babel src --out-dir lib'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the react-a11y package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     babel src --out-dir lib
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs react-a11y
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls react-a11y
> npm ERR! There is likely additional logging output above.
> npm WARN Local package.json exists, but node_modules missing, did you mean to install?
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /Users/mhuggins/Development/kapost/pepperweed/node_modules/react-a11y/npm-debug.log
> npm ERR! Darwin 15.4.0
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "romeovs/react-a11y" "--save"
> npm ERR! node v5.11.0
> npm ERR! npm  v3.8.6
> npm ERR! code ELIFECYCLE
> 
> npm ERR! react-a11y@1.0.0 postinstall: `npm run build`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the react-a11y@1.0.0 postinstall script 'npm run build'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the react-a11y package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     npm run build
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs react-a11y
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls react-a11y
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /Users/mhuggins/Development/kapost/pepperweed/npm-debug.log

I'm running fairly recent versions of node & npm:

> ➜  node -v
> v5.11.0
> ➜  npm -v 
> 3.8.6

**Comment by mhuggins** [comment]
I ended up having to run all of the following before this package would install properly:

> npm install babel-cli --save-dev
> npm install babel-plugin-transform-runtime --save-dev
> npm install babel-preset-stage-0 --save-dev
> npm install romeovs/react-a11y --save

It's likely that I would have needed to do the same for `babel-preset-es2015` and `babel-preset-react` as well if they weren't already in my package.json.

With that said, I think it would make sense to either include these as `dependencies` or `peerDependencies` instead of `devDependencies` in react-a11y's package.json.  Without doing so, this will continue to be a problem for users in the future.

**Comment by jkomusin** [comment]
I believe this is simply caused by #7, the `postinstall` step is a duplication of the `prepublish` step.

**Comment by erin-doyle** [comment]
I think this has been resolved or at least can be closed as a duplicate to #7 .  I can install successfully but there are still other issues (see #1  ).
